src/api/Readme_rubocop.md

```diff
@@ -0,0 +1,11 @@
+# Fixing rubocop styling issues
+
+* Pick one rubocop rule and enable it in .rubocop.yml
+
```

Comment:
I would propose not to touch .rubocop_todo.yml at all, but run 'rubocop --auto-gen-config' after each step. That task updates the .rubocop_todo.yml.